dev/ColorPicker/ColorPicker.idl

```diff
@@ -45,6 +45,8 @@ unsealed runtimeclass ColorPicker : Windows.UI.Xaml.Controls.Control
     [MUX_DEFAULT_VALUE("winrt::ColorSpectrumComponents::HueSaturation")]
     ColorSpectrumComponents ColorSpectrumComponents { get; set; };
     event Windows.Foundation.TypedEventHandler<ColorPicker, ColorChangedEventArgs> ColorChanged;
+    [MUX_DEFAULT_VALUE("winrt::Orientation::Vertical")]
```

Comment:
Can you add this under MUX_PUBLIC_V2  to keep it ABI compatible ? [https://github.com/microsoft/microsoft-ui-xaml/blob/0db5d0398cb38d69b06e26ef734cbbdebdebe774/idl/Microsoft.UI.Xaml.idl#L179](https://github.com/microsoft/microsoft-ui-xaml/blob/0db5d0398cb38d69b06e26ef734cbbdebdebe774/idl/Microsoft.UI.Xaml.idl#L179)

@jevansaks as FYI to double check.

Reply:
Yes please. @ranjeshj do you have an ETA on the ABI compatibility test addition? Maybe now that WinUI 2.6 is out the door we can take a little time for that test?

We also need to change the framework package versioning scheme for 2.7 (there's a github issue tracking that, if there's a way for you to tag that on the "do this soon" that would be great. 🙂)

Reply:
Done!